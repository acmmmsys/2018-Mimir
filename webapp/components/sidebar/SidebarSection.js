import React, {Component} from 'react';
import classnames from 'classnames';
import 'style/sidebar/SideBar.scss';

class SidebarSection extends Component {
    render() {
        return (
            <section className='ui-sidebar-section'>
                <header className='ui-sidebar-section-header'>
                    { this.props.title }
                </header>
                { this.props.children }
            </section>
        )
    }
}

export default SidebarSection;