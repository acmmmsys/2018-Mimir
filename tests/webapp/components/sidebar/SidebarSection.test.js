import React from 'react';
import SidebarSection from '../../../../webapp/components/sidebar/SidebarSection'
import renderer from 'react-test-renderer';

let wrapper;

test('SidebarSection does render', () => {
    wrapper = shallow(
        <SidebarSection>

        </SidebarSection>
    );
    expect(wrapper).toMatchSnapshot();
});
