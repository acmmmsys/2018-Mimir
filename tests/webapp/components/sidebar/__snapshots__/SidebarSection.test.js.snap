// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarSection does render 1`] = `
<section>
  <header />
</section>
`;
